src/docs/contributing.mdx

```diff
@@ -92,131 +75,7 @@ Supported hardware for PHDI tools includes:
 - Windows 10/11
 - Mac (Intel or Apple silicone)
 
-#### Software
-
-##### Overview
```

Comment:
elected to delete most of this (now obsolete?) information that was written for the original PHDI setup. Can add it back if folks think it's still useful / relevant